src/main/java/bio/terra/cli/command/shared/WsmBaseCommand.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.cli.command.shared;
+
+import bio.terra.cli.app.utils.VersionCheckUtils;
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.User;
+import bio.terra.cli.command.Main;
+import bio.terra.cli.utils.Logger;
+import bio.terra.cli.utils.UserIO;
+import java.io.PrintStream;
+import org.slf4j.LoggerFactory;
+
+public abstract class WsmBaseCommand extends BaseCommand {
+  private static final org.slf4j.Logger logger = LoggerFactory.getLogger(WsmBaseCommand.class);
+  protected static PrintStream OUT;
+  protected static PrintStream ERR;
+
+  @Override
+  public Integer call() {
```

Comment:
Can 18-51 be replaced with

```
  @Override
  public Integer call() {
    // Check if this version of the CLI is out of date
    if (VersionCheckUtils.isObsolete()) {
      ERR.printf(
          "Warning: Version %s of the CLI has expired. Functionality may not work as expected. To install the latest version: curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/download-install.sh | bash ./terra\n"
              + "If you have added the CLI to your $PATH, this step will need to be repeated after the installation is complete.%n",
          bio.terra.cli.utils.Version.getVersion());
      return 0;
    }

    return super.call();
  }
```

Reply:
Wow. Thank you, Melissa. Just a heads-up, It also need necessary initialization before calling VersionCheckUtils, so I add L21-25. 
I also keep the L26-27 with the L13. How do you think it? if we do not need this logger, I can remove these lines.